plugins/eslint-plugin-liferay-portal/lib/rules/no-explicit-extend.js

```diff
@@ -60,6 +63,7 @@ function fix(nodesToRemove, context, fixer) {
 
 	// Record index of last `item` in `items` that will remain after we've
 	// deleted `nodesToRemove`.
+
```

Comment:
looks so odd to have a line after in-line comments. Almost as if the code it was commenting was deleted. but then again, its all arbitrary 🤷‍♂️